CMakeLists.txt

```diff
@@ -320,7 +320,10 @@ endif()
 # On Android RELEASE builds, we disable exceptions and RTTI to save some space (about 75 KiB
 # saved by -fno-exception and 10 KiB saved by -fno-rtti).
 if (ANDROID OR IOS OR WEBGL)
-    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -fno-rtti")
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-exceptions -fno-rtti")
+    if (ANDROID OR WEBGL)
+        set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fno-unwind-tables -fno-asynchronous-unwind-tables")
```

Comment:
`-fno-unwind-tables` explain why there were no stack traces. Although without it the build still relies on toolchain defaults. Is `-funwind-tables` the default in case of iOS and macOS. Can you find the `__eh_frame` and/or `__unwind_info` sections in the binary (on x86) when no unwind tables related flag is specified? What about the size of the final binary?

Reply:
Interestingly Crashlytics is not using libunwind but some custom solution, e.g. parses the image by hand looking for the indicated sections.
https://github.com/firebase/firebase-ios-sdk/blob/bdac2b87f6d6e604deae914a970a75136b3a62b5/Crashlytics/Crashlytics/Components/FIRCLSBinaryImage.m#L340-L352

Reply:
> Although without it the build still relies on toolchain defaults

I thought about this too. I figured if I test it x64/arm64 iPads and macs, then I would accept relying on the toolchain defaults.

> Can you find the __eh_frame and/or __unwind_info sections in the binary

I could find `__eh_frame`, but not `__unwind_info`.
`objdump -h libbackend.a (x86_64 arch)`:
```python
Sections:
Idx Name             Size     VMA              Type
  0 __text           000000fc 0000000000000000 TEXT
  1 __objc_methname  00000076 00000000000000fc DATA
  2 __objc_selrefs   00000028 0000000000000178 DATA
  3 __bitcode        00001550 00000000000001a0 DATA
  4 __cmdline        0000020f 00000000000016f0 DATA
  5 __objc_imageinfo 00000008 00000000000018ff DATA
  6 __compact_unwind 00000020 0000000000001908 DATA
  7 __eh_frame       00000058 0000000000001928 DATA
```

For comparison here is what you can see in an object file in CommonApp. There is `__eh_frame`, but no `__unwind_info` either.
`objdump -h RayPicking.o`:
```python
Sections:
Idx Name             Size     VMA              Type
  0 __text           00006da1 0000000000000000 TEXT
  1 __literal16      00000140 0000000000006db0 DATA
  2 __literal8       000000d0 0000000000006ef0 DATA
  3 __gcc_except_tab 00000090 0000000000006fc0 DATA
  4 __literal4       00000004 0000000000007050 DATA
  5 __cstring        000000cc 0000000000007054 DATA
  6 __const          00000073 0000000000007120 DATA
  7 __const          00000140 0000000000007198 DATA
  8 __compact_unwind 00000520 00000000000072d8 DATA
  9 __eh_frame       00000720 00000000000077f8 DATA
```

`objdump -u` lists a bunch of `__compact_unwind` sections **for both** Filament object files and object files of CommonApp, so I think it is fair to say that unwind info is indeed generated with LLVM 13.0.0.

Reply:
The __bitcode section in the Filament lib is a bit concerning, since AFAIK we don't embed bitcode in any of our own libs. I will discuss this on Slack with you whether bitcode could cause problems here or not

Reply:
> What about the size of the final binary?

Added the answer to the PR description. It didn't grow much.